### PR TITLE
Added commandlet for oracle remote restore.

### PR DESCRIPTION
**Pull request by KavishreeShanmugam11**

### CODE REVIEW

File: src/Cohesity.Powershell/Cohesity.PowerShell.psd1 [hunk 1]
```diff
@@ -132,11 +132,12 @@ FunctionsToExport = @(
                     'Remove-CohesityVirtualIP',
                     'Remove-CohesityVlan',
                     'Restore-CohesityBackupToView',
-                    'Restore-CohesityFileV2',                   
+                    'Restore-CohesityFileV2',
+                    'Restore-CohesityOracleDatabase',
                     'Restore-CohesityRemoteFile',
                     'Restore-CohesityRemoteFileV2',
                     'Restore-CohesityRemoteMSSQLObject',
-                    'Restore-CohesityOracleDatabase',
```
Comment (pyashish) [inline src/Cohesity.Powershell/Cohesity.PowerShell.psd1:139]: This breaks compatibility?

**Review by pyashish — CHANGES_REQUESTED** [review]

File: src/Cohesity.Powershell/Cohesity.PowerShell.psd1 (hunk 1 above)
Comment (KavishreeShanmugam11) [reply src/Cohesity.Powershell/Cohesity.PowerShell.psd1:139]: No. Just arranged the commandlet ahabetically.

Comment (pyashish) [reply src/Cohesity.Powershell/Cohesity.PowerShell.psd1:139]: My bad. didnt see it correctly.

File: src/Cohesity.Powershell/Scripts/Restore/Restore-CohesityRemoteOracleDatabase.ps1 [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        .SYNOPSIS
+        Restores the specified files or folders from a remote backup.
+        .DESCRIPTION
+        Restores the specified files or folders from a remote backup. This commandlet supports only source with environment type VMware/Physical/Isilon.
+        .NOTES
+        Published by Cohesity.
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Restore-CohesityRemoteOracleDatabase -TaskName "Restore-Oracle-Object" -FileName /C/data/file.txt -JobId 1234 -SourceId 843 -TargetSourceId 856 -RestoreMethod AutoDeploy -TargetVMCredential (Get-Credential)
```
Comment (pyashish) [inline src/Cohesity.Powershell/Scripts/Restore/Restore-CohesityRemoteOracleDatabase.ps1:12]: Mention how to get other numbers , from which cmdlet etc.

**Review by pyashish — APPROVED** [review]
Minor nits, incoporate the changes and push.

**Pull request merged**